packages/vats/src/bridge.js

```diff
@@ -48,6 +55,9 @@ const prepareScopedManager = zone => {
       inboundHandler,
     }),
     {
+      getBridgeId() {
+        return this.state.bridgeId;
+      },
```

Comment:
This would require an upgrade of the bridge vat, which I believe we're not planning on?

Reply:
Great point! I'll make it optional so nothing depends on it being there.